src/OpenTelemetry/Instrumentation/PropertyFetcher.cs

```diff
@@ -13,13 +13,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
+
 using System;
 using System.Linq;
 using System.Reflection;
 
-namespace OpenTelemetry.Instrumentation.Http.Implementation
+namespace OpenTelemetry.Instrumentation
 {
-    internal class PropertyFetcher
+    public class PropertyFetcher
```

Comment:
nit: Should have XML comments now that it is public.

Reply:
added some. not sure if the best text...